### ISSUE TITLE
Site Owner User Story: Automatically Profile creation

### ISSUE THREAD

**Issue by DMASCoreDeclan** [DMASCoreDeclan/PRD-PP4-Blog#36]
As the **Site Owner** 
I can **ensure User Profiles have a default image** 
so that **a User has a default Profile that they can change in profile.html when they are logged in**

At presend users/models appends a CloudinaryField to the User model.  When used from admin/users/profile/add/, you can create a Profile which loads a default image if no specific image is added.  The default image, 'placeholder.jpg' is stored in Cloudinary.  However the profile is currently created by a manual process, I want to automate this.

- Acceptance criteria 1
placeholder.jpg is placed in the profile and displayed in profile.html once the user registration process is complete

- Acceptance criteria 2
Ensure this code is no longer needed
{% if user.profile.profile_picture.url  %}
            <img class="rounded-circle account-img" src="{{ user.profile.profile_picture.url }}>
        {% else %}
            img class="rounded-circle account-img" src=" https://res.cloudinary.com/dlzmqu24r/image/upload/v1706613373/placeholder.jpg  
        {% endif %}

**Comment by DMASCoreDeclan** [comment]
**Acceptance criteria 1**
A new User Wendy was created and their Profile was also created:
![image](https://github.com/DMASCoreDeclan/PRD-PP4-Blog/assets/54984695/09da765e-92f1-4f18-904f-83d0e9a6d4ee)


**Acceptance criteria 2**
Previously, this code was required:
![image](https://github.com/DMASCoreDeclan/PRD-PP4-Blog/assets/54984695/fba88395-52d0-468e-88e5-1b2fd3555ac6)

However, the code now produces the same result when the DTL is removed:
![image](https://github.com/DMASCoreDeclan/PRD-PP4-Blog/assets/54984695/d3c2a8be-fa2a-422b-957d-3a0b909038c2)
